src/index.ts

```diff
@@ -242,6 +235,33 @@ class Marian {
     res.writeHead(200, headers);
     res.end(JSON.stringify(response));
   }
+
+  async load() {
+    let taxonomy: Taxonomy;
+    try {
+      taxonomy = await this.fetchTaxonomy(process.env.TAXONOMY_URL!);
```

Comment:
We should check if the `TAXONOMY_URL` env var is undefined instead of non-null assertion.

Reply:
there actually is no taxonomy url at the moment, hence the placeholder data. i will add a comment here to update this into the env variable verification i included above